Source/IGListCollectionView.m

```diff
@@ -11,25 +11,6 @@
 
 @implementation IGListCollectionView
 
-- (void)commonInit {
-    self.backgroundColor = [UIColor whiteColor];
-    self.alwaysBounceVertical = YES;
-}
-
-- (instancetype)initWithFrame:(CGRect)frame collectionViewLayout:(UICollectionViewLayout *)layout {
-    if (self = [super initWithFrame:frame collectionViewLayout:layout]) {
-        [self commonInit];
```

Comment:
FWIW the rationale for removal was that `IGListCollectionView` exists only to provide the compiler warnings but I'm more than happy to update the PR, I agree it's convenient.